srcs/layouts/shaw_imperial_en.xml

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Shaw Imperial layout for shavian alphabet, see https://www.shavian.info/keyboards/ -->
+<keyboard name="Shaw Imperial" script="shavian">
```

Comment:
The numpad and the optional number row (that can be enabled in the settings) can be configured to match the layout's script by modifying this function: https://github.com/Julow/Unexpected-Keyboard/blob/master/srcs/juloo.keyboard2/KeyModifier.java#L100

Reply:
Hmm, I don't think I understand. If I got it correctly, this function converts regular numbers to their equivalents in a certain script, but shavian doesn't have any special numbers of its own, they stay the same.

Reply:
My mistake!